app/src/componentsV2/modals/CreateWorkspaceModal/components/LocalWorkspaceCreationView/LocalWorkspaceCreationView.tsx

```
@@ -0,0 +1,198 @@
+  return (
+    <div key={item.path} className={`preview-folder-items__item ${isNewWorkspace ? "new-workspace-item" : ""}`}>
+      <div className="preview-folder-item-content">
+        <Icon className="preview-folder-item-icon" />
+        <span className="preview-folder-item-name">{item.name}</span>
+      </div>
+    </div>
+  );
+};
+
+// eslint-disable-next-line no-control-regex
+const INVALID_FS_NAME_CHARACTERS = /[<>:"/\\|?*\x00-\x1f]/g;
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)